index.html

```diff
@@ -0,0 +1,135 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>RikolPortfolio</title>
+        <link rel="stylesheet" href="style.css" />
+    </head>
+    
+    <body>
+        <header id="header">
+            <h3 id="logo" class="content-padding"><a href="#intro">My Logo</a></h3>
```

Comment:
- [optional] since it wasnt mention in the previous review, please ensure that your close icon is showing. it works fine
 
![Screenshot (519)_LI](https://user-images.githubusercontent.com/65298048/195403284-8f5b55b9-ae40-44ec-ba69-825cfd112548.jpg)